frontend/__tests__/components/phone-number.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen, waitFor } from '@testing-library/react';
+
+import { describe, expect, it } from 'vitest';
+
+import { PhoneNumber } from '~/components/phone-number';
+
+describe('<PhoneNumber />', () => {
+  it('Renders component with phone number', async () => {
+    render(<PhoneNumber phoneNumber="1234567890" />);
+    const element: HTMLInputElement = await waitFor(() => screen.findByTestId('phoneNumber'));
```

Comment:
Is the `waitfor(..)` required here? This seems to work but I'm not sure if it's working by accident:

``` typescript
const element = await screen.findByTestId<HTMLInputElement>('phoneNumber')
```

Reply:
I guess not.  Anyways, removed from all instances.